### ISSUE TITLE
Add comment

### ISSUE THREAD

**Issue by orianna-zzo** [orianna-zzo/AllinOne#12]
How about adding Gitalk?

**Comment by devcui** [comment]
as a developer so i want to use Gitalk,but if i was a writter  may be i will choose livere

**Comment by alexpov** [comment]
@orianna-zzo , yes, it should nice to have a comments section for posts. Super important when managing a blog :)

**Comment by alexpov** [comment]
[0001-added-disqus-support-for-post-commenting.zip](https://github.com/orianna-zzo/AllinOne/files/2928597/0001-added-disqus-support-for-post-commenting.zip)

@orianna-zzo I prepared a commit that adds support for disqus ... feel free to apply (couldn't push a PR since the repo is blocked)
